server/src/triggers/eventTriggers/device.onDeviceNameChanged/post.mjs

```diff
@@ -0,0 +1,23 @@
+/*
+* Copyright 2021 Comcast Cable Communications Management, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+function post(ctx) {  
+ if( ctx.result ){
```

Comment:
Please do the console log no matter what (so it always happens). Then either include ctx.result in what you log or log a second statement if it is truthy.

Reply:
added logger in post.mjs for ctx.result